hw/ip/pattgen/data/pattgen.hjson

```diff
@@ -0,0 +1,144 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+{ name: "pattgen"
+  clock_primary: "clk_fixed"
+  bus_device: "tlul"
+  bus_host: "none"
+  estimated_gate: "1k"
+  // OUTPUT pins
+  available_output_list: [
+    { name: "pda1_tx", desc: "Serial output data bit for pattern generation on channel 1" }
+    { name: "pcl1_tx", desc: "Bit-banging output clock for pattern generation on channel 1" }
```

Comment:
do we actually need both the data and clock...?

Reply:
also i think the original one pager calls for 2 groups of 2, so there should be 4 total here..i imagine something like 

```
pda1_tx0
pda1_tx1
pda2_tx0
pda2_tx1
```

Reply:
@tjaychen i think clk and data should be used in pair